src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -488,8 +488,21 @@ def subset_results_by_state(processed_results_dir: str, state: str) -> None:
     abbrev_name_dict = {v: k for k, v in metadata.US_STATE_ABBRV_MAP.items()}
     state_name = sanitize_location(abbrev_name_dict[state.upper()])
     processed_results_dir = Path(processed_results_dir)
-    usa_results_dir = processed_results_dir / "usa"
-    state_dir = processed_results_dir / "states" / state_name
+    dirs = [x for x in processed_results_dir.iterdir() if x.is_dir()]
+    # Get USA results directory
+    for d in dirs:
```

Comment:
I think this all makes the assuption that there is only one directory in this folder, correct? That may  not be the case for various reasons.

This may be tougher than I thought.

To get the version number you'll need to sort dirnames and pick the last one (b/c 1.0.1 > 1.0.0).

But I'm not sure offhand how that will work for runs done without --version.

Reply:
There will only be one "USA" directory in here as it is timestamp when you run make results.